CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This is a new major version that contains several backwards-compatibility breaks
 
 * #1309 Drop ESLint integration (@Kocal) 
 
+* #1318 Drop webpack-dev-server 4 support, only webpack-dev-server 5 is supported (@Kocal)
```

Comment:
I suggest mentioning that the supported option in the configuration callback have changed, with a link to their changelog. They have BC breaks in their configuration.

Reply:
Done